.github/workflows/ci.yml

```
@@ -0,0 +1,30 @@
+          enable-cache: true
+      - run: uv python install
+      - name: requirements.txt
+        run: |
+          if ! uv export --frozen --no-hashes --format requirements-txt | cmp -s requirements.txt -; then
+            echo "Re-run uv export --frozen --no-hashes --format requirements-txt > requirements.txt"
+            exit 1
+          fi
+      - run: uv sync --frozen
+      - name: pytest
+        run: uv run --frozen pytest
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)